src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/_params.py

```diff
@@ -30,7 +32,7 @@ def load_arguments(self, _):
         c.argument('password', options_list=['-p', '--password'], help='The Microsoft account (MSA) password for the bot.')
         c.argument('storageAccountName', options_list=['-s', '--storage'], help='Storage account name to be used with the bot. If not provided, a new account will be created.', arg_group='Web/Function bot Specific')
         c.argument('tags', arg_type=tags_type)
-        c.argument('language', help='The language to be used to create the bot.', options_list=['--lang'], arg_type=get_enum_type(['Csharp', 'Node']), arg_group='Web/Function bot Specific')
+        c.argument('language', help='The language to be used to create the bot.', options_list=['--lang'], arg_type=get_enum_type(supported_languages), arg_group='Web/Function bot Specific')
```

Comment:
This options list is fine but it isn't needed. Argparse will recognize `--lang` as matching `--language`.

Reply:
Oh I didn't know that. Thanks for the hint.

Reply:
I'll leave it as-is since it doesn't seem to be a problem. Will keep in mind for the future.